packages/core/useSpeechRecognition/index.md

```diff
@@ -34,3 +34,27 @@ The following shows the default values of the options, they will be directly pas
   continuous: true,
 }
 ```
+
+You can also bind the lang property to a ref object. Note that updating this ref value cannot take effect until the next time the recognition is stopped.
```

Comment:
Have arguments accepts ref is a common practice in VueUse, I don't think we need to explicitly document it here. (but thanks!)